### ISSUE TITLE
CoreWidgetProcessing (in)visibilityConfig should allow subscription to topics from other scopes

### ISSUE THREAD

**Issue by AFaust** [Alfresco/Aikau#1097]
This is primarily to outsource a side-comment I made in the description of #1096. Similar to the AlfDynamicPayloadButton the CoreWidgetProcessing (in)visibilityConfig should allow subscription to topics from different scopes than the widget scope. That is, it should be possible to configure rules like:

```javascript
{
   name: "alfresco/html/Label",
   config: {
      label: "Label",
      visibilityConfig: {
         initialValue: true,
         rules: [{
            // either topic or targetId is required (after #1047)
            topic : 'MY_SUBSCRIPTION_TOPIC',
            // subscribeGlobal true/false = ultimate preference over subscribeParent, custom scope and default widget scope
            subscribeGlobal : true,
            // subscribeParent true/false = supercedes custom scope and default widget scope
            subscribeParent : true,
            // subscribeScope = alternative to default widget scope
            attribute: 'value',
            is: [ "1" ],
            strict: true
         }]
      }
   }
}
```

This would be useful when an "outer" widget needs to react to publications that originate from a nested, complex widget (like a form) that has a custom pubSubScope set.

**Comment by draperd** [comment]
I've raised https://issues.alfresco.com/jira/browse/AKU-988 to address this. I've planned it for the next sprint (but you should be aware that due to team vacation that this sprint is unlikely to start next week, rather it may be delayed by a week). However, we will bring the issue forwards if we can - I just wanted to set reasonable expectations

**Comment by draperd** [comment]
I've had to bump this from Sprint 72 unfortunately... there is actually *no-one* working full-time on Aikau in the current sprint, but I'm going to get bugs and features required for Alfresco teams done - if I can I might try and pull it into the Sprint depending on fast I progress my other work.

**Comment by draperd** [comment]
This is in plan for Sprint 73... so should be available in the next release (1.0.73)

**Comment by draperd** [comment]
This is now resolved in the 1.0.73 release that is now available
